internal/uploader/auth_utils_test.go

```diff
@@ -0,0 +1,141 @@
+package uploader
+
+import (
+	"fmt"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/models"
+	"github.com/openshift/assisted-service/pkg/k8sclient"
+	"gorm.io/gorm"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var _ = Describe("extractToken", func() {
+
+	It("successfully extracts a token from a valid pull secret", func() {
+		pullSecret := `{"auths":{"cloud.openshift.com":{"auth":"dXNlcjpwYXNzd29yZAo=","email":"r@r.com"}}}`
+		token, err := extractToken([]byte(pullSecret), "cloud.openshift.com")
+		Expect(err).NotTo(HaveOccurred())
+		Expect(token).To(Equal("dXNlcjpwYXNzd29yZAo="))
+	})
+	It("fails to extract a token from an invalid pull secret", func() {
+		pullSecret := `not a pull secret`
+		token, err := extractToken([]byte(pullSecret), "cloud.openshift.com")
+		Expect(err).To(HaveOccurred())
+		Expect(token).To(BeEmpty())
+	})
+	It("fails to extract a token when the pull secret doesn't have the right entry", func() {
+		pullSecret := `{"auths":{"clouds.openshift.com":{"auth":"dXNlcjpwYXNzd29yZAo=","email":"r@r.com"}}}`
+		token, err := extractToken([]byte(pullSecret), "cloud.openshift.com")
+		Expect(err).NotTo(HaveOccurred())
+		Expect(token).To(BeEmpty())
+	})
+	It("fails to extract a token when the pull secret doesn't have a token for the entry", func() {
+		pullSecret := `{"auths":{"clouds.openshift.com":{"email":"r@r.com"}}}`
```

Comment:
This looks like it should be `cloud.openshift.com` for this test to be valid. Otherwise it's just the same as the previous one, right?

Reply:
Oops, right! Thanks for catching this! Accidental copy-paste error 😅